os/board/common/crashdump.c

```diff
@@ -167,6 +358,35 @@ static int ramdump_via_uart(void)
  ****************************************************************************/
 
 #if defined(CONFIG_BOARD_CRASHDUMP)
+void crashdump_init(void)
+{
+	int index;
+
+	index = get_dump_partindex("coredump");
+	if (index < 0) {
+		lldbg("partition index is wrong for coredump part name\n");
+		return;
+	}
+	snprintf(coredump_partname, MTD_PARTNAME_LEN + 1, "/dev/mtdblock%d", index);
```

Comment:
It looks unused. Could you let me know why do you save this partname?

Reply:
It will be used for reading through mtd layer by tools in future.